### PR TITLE
NO-JIRA: testing revbump at startup

### PR DESCRIPTION
**Pull request by tjungblu**
/hold

### CODE REVIEW

File: server/revbump/revbump.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	lg.Info(
+		"bumping latest revision",
+		zap.Int64("latest-revision", latest.main),
+		zap.Int64("bump-amount", amount),
+		zap.Int64("new-latest-revision", latest.main+amount),
+	)
+
+	latest.main += amount
+	latest.sub = 0
+	k := make([]byte, 17)
```
Comment (dusk125) [inline server/revbump/revbump.go]: nit: Could do revBytesLen instead of 17

**Review by dusk125 — COMMENTED** [review]
One nit otherwise lgtm

File: server/revbump/revbump.go (hunk 1 above)
Comment (tjungblu) [reply server/revbump/revbump.go]: good catch, I was just copy and pasting it over to make the dependency graph happy

Comment (tjungblu) [reply server/revbump/revbump.go]: fixed

**Pull request closed without merging**